File: from-2101-to-2200/2161-partition-array-according-to-given-pivot/solution.py
class Solution:
    def pivotArray(self, nums: List[int], pivot: int) -> List[int]:
        lt, eq, gt = [], [], []
        for num in nums:
            if num < pivot:
                lt.append(num)
            elif num > pivot:
                gt.append(num)
            else:
                eq.append(num)
                
        return lt + eq + gt
        

File: from-2101-to-2200/2161-partition-array-according-to-given-pivot/solution.go
func pivotArray(nums []int, pivot int) []int {
    lt, eq, gt := make([]int, 0), make([]int, 0), make([]int, 0)
    for _, num := range(nums) {
        if num < pivot {
            lt = append(lt, num)
        } else if num > pivot {
            gt = append(gt, num)
        } else {
            eq = append(eq, num)
        }
    }
    return append(lt, append(eq, gt...)...)
}